docs/modules/mongodb_watchdog.md

```diff
@@ -11,16 +17,16 @@ This module uses a separate database, using the `logger` database alias in setti
 
  * `mongodb.watchdog.limit`
     * the maximum severity level (0 to 7, per RFC 5424) to save into watchdog.
-    * Errors below this level (with a higher numerical level) will be ignored by the module.
-    * If not defined, all events will saved.
+    * errors below this level (with a higher numerical level) will be ignored by the module.
```

Comment:
actually, below the ERROR level, these are no longer errors, but warnings, information... I think we use the term "event" in the code and the rest of the page for these, not "errors".